lms/djangoapps/teams/tests/test_views.py

```diff
@@ -966,6 +966,37 @@ def test_delete_removed_from_search(self):
             user='staff'
         )
 
+    def test_duplicates_and_nontopic_private_teamsets(self):
+        # create a team in a private teamset and add a user
+        unprotected_team_in_private_teamset = CourseTeamFactory.create(
+            name='unprotected_team_in_private_teamset',
+            description='unprotected_team_in_private_teamset',
+            course_id=self.test_course_1.id,
+            topic_id='private_topic_1_id',
+        )
+        unprotected_team_in_private_teamset.add_user(self.users['student_enrolled'])
+
+        # make some more users and put them in the solar team.
+        self.create_and_enroll_student(username='another_student')
```

Comment:
Trivial nit, but I have been trying to use variables when a string is used more than once i.e:
```
another_student_username = 'another_student'
self.create_and_enroll_student(username=another_student_username)
```
To cut on typos.